packs/criticaldeck/critical-fumble-deck-1.json

```diff
@@ -13,7 +13,7 @@
             "src": null,
             "system": {},
             "text": {
-                "content": "<section class=\"fumble-deck\">\n<h1>Meant to do That</h1>\n<blockquote>\n<p>You are moved 10 feet in a random direction (determined by the GM). This movement triggers reactions.</p>\n</blockquote>\n<p><code>Melee</code></p>\n<h1>Misjudged the Distance</h1>\n<blockquote>\n<p>Until the end of your next turn, all your range increment penalties are doubled.</p>\n</blockquote>\n<p><code>Ranged</code></p>\n<h1>Not the Weak Point</h1>\n<blockquote>\n<p>You take and [[/r 1d6[bleed]]] and can't use this attack until the end of your next turn.</p>\n</blockquote>\n<p><code>Unarmed</code></p>\n<h1>How did that Happen?</h1>\n<blockquote>\n<p>You call forth a mist with the effects of <em>@UUID[Compendium.pf2e.spells-srd.Item.Stinking Cloud]</em> centered on a corner of your space (determined by the GM).</p>\n</blockquote>\n<p><code>Spell</code></p>\n<p> </p>\n</section>",
```

Comment:
This isn't appropriate: the actor rolling is the actor receiving bleed damage. They don't have abilities that modify it.